bindings/mqtt/mqtt.go

```diff
@@ -204,8 +204,13 @@ func (m *MQTT) Invoke(req *bindings.InvokeRequest) (*bindings.InvokeResponse, er
 	bo = backoff.WithContext(bo, m.ctx)
 
 	return nil, retry.NotifyRecover(func() error {
-		m.logger.Debugf("mqtt publishing topic %s with data: %v", m.metadata.topic, req.Data)
-		token := m.producer.Publish(m.metadata.topic, m.metadata.qos, m.metadata.retain, req.Data)
+		topic, ok := req.Metadata[mqttTopic]
```

Comment:
Please add test for this case.

Reply:
OK, thanks.

Reply:
Hi, I just pushed two tests related to the commit I made.